CourseApp/RPGSaga/GameBuilder/FightBuilder.cs

```diff
@@ -0,0 +1,49 @@
+namespace CourseApp.RPGSaga.GameBuilder
+{
+    using CourseApp.RPGSaga.Heroes;
+
+    public class FightBuilder
+    {
+        private Player _p1;
+        private Player _p2;
+        private Player _winner;
+        private Player _looser;
+
+        public FightBuilder(Player p1, Player p2)
+        {
+            _p1 = p1;
+            _p2 = p2;
+        }
+
+        public void SetTragets()
+        {
+            _p1.SetTarget(_p2);
+            _p2.SetTarget(_p1);
+        }
+
+        public Player StartFight()
+        {
+            Logger.WriteLog($"Fight: [{_p1.ToString()} {_p1.Hp} HP {_p1.Strength} Strength] against [{_p2.ToString()} {_p2.Hp} HP {_p2.Strength} Strength]");
```

Comment:
:no_bell: `.ToString()` is redundant here, template strings invokes it automaticly

```suggestion
            Logger.WriteLog($"Fight: [{_p1} {_p1.Hp} HP {_p1.Strength} Strength] against [{_p2} {_p2.Hp} HP {_p2.Strength} Strength]");
```

Reply:
Spravedlivo